app.rb

```diff
@@ -0,0 +1,29 @@
+require 'sinatra/base'
+require_relative 'lib/database_connection_setup'
+require_relative 'lib/peep'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    @peeps = Peep.all
+
+    erb :index
+  end
+
+  post '/' do
+    @peeps = Peep.all
+    message = params['message']
+
+    @zero_length_msg = message.length.zero?
+
+    unless @zero_length_msg
+      DatabaseConnection.query("INSERT INTO peeps VALUES (DEFAULT, DEFAULT, '#{message}')")
+      redirect '/'
+    end
+
+    erb :index
```

Comment:
not necessary to put the erb :index line in 'post' methods

Reply:
True, and yet it was the best solution I could come up with at the time... I need it to load the `index.erb` template so that it can respond to the true/false state of `@zero_length_msg`. This was in order to achieve the effect of `Sinatra/Flash` which I was unable to get working (with `enable :sessions`)

Reply:
Assuming the "happy path", program flow would go to `redirect '\'` as we would expect the `post` to do. I think maybe if I could get Sinatra/Flash working then I wouldn't have use `erb :index` in this `post` method.